src/main/java/at/hannibal2/skyhanni/config/features/rift/area/mountaintop/SunGeckoConfig.java

```diff
@@ -0,0 +1,42 @@
+package at.hannibal2.skyhanni.config.features.rift.area.mountaintop;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import at.hannibal2.skyhanni.config.core.config.Position;
+import com.google.gson.annotations.Expose;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigLink;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;
+
+public class SunGeckoConfig {
+
+    @Expose
+    @ConfigOption(name = "Enabled", desc = "Show Sun Gecko Helper.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean enabled = true;
+
+    @Expose
+    @ConfigOption(name = "Show Modifiers", desc = "Show a list of modifiers in the overlay.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean showModifiers = false;
+
+    @Expose
+    @ConfigOption(name = "Highlight Real Boss", desc = "Highlights the real boss in green.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean highlightRealBoss = false;
+
+    @Expose
+    @ConfigOption(name = "Highlight Clones", desc = "Highlights the fakes bosses in red.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean highlightFakeBoss = true;
```

Comment:
i dont think every single option here should have a FeatureToggle

Reply:
didnt even notice was just copy and pasting tbh